src/Image/Image.js

```diff
@@ -276,7 +284,7 @@ module.exports = kind(
 	*/
 	handleLoad: function () {
 		if (!this.sizing && this.placeholder) {
-			this.applyStyle('background-image', null);
+			this._errorHandling? this._errorHandling = false : this.applyStyle('background-image', null);
```

Comment:
I'd rather see an if/else block here rather than the ternary operator for clarity.